docs/latest/getting-started/configuring-auth.md

```diff
@@ -0,0 +1,82 @@
+---
+description: |
+  Add authentication to your project, by using Fresh's powerful auth system. 
+---
+
+Authentication and session management are essential for building modern web
+applications, enabling you to hide premium content or create admin-only sections
+like dashboards. Fresh is an edge-native web framework that embraces progressive
+enhancement through server-side rendering and islands architecture while
+optimizing for latency and performance. As a result, Fresh apps tend to achieve
+higher Lighthouse scores and can function in areas with low internet bandwidth.
+
+This guide covers the different ways to add authentication into your Fresh app,
+emphasizing plugins, and integrating with other systems. Here are some examples:
+
+## Basic Auth Plugin
+
+A [basic auth plugin](https://github.com/hashrock/hashrock-fresh-plugins) (which
+should only be used in your development environment) can be added to your Fresh
+app like this:
+
+```ts main.ts
+import basicAuthPlugin from "https://deno.land/x/hashrock_fresh_plugins/basic.ts";
+
+await start(manifest, {
+  plugins: [basicAuthPlugin("/greet/")],
+});
+```
+
+To use this plugin, create a .env file:
+
+```.env
+BASIC_AUTH_USER=user 
+BASIC_AUTH_PASSWORD=password
+```
+
+## Deno KV Auth
+
+For a look at how you can manually bring Deno KV Auth into your project, see
+[this example](](/docs/examples/using-deno-kv-oauth.md).
+
+To quickly configure your application using a Deno Fresh plugin, you can see the
+documentation for the
+[Deno Fresh KV Auth plugin](https://deno.land/x/deno_fresh_kv_oauth).
```

Comment:
Deno KV OAuth will soon be getting an official Fresh plugin. Can you please hold off on this part? 🙂 

Ref: https://github.com/denoland/deno_kv_oauth/pull/171

Reply:
The Fresh plugin has been added to Deno KV OAuth. Check out the docs here: https://github.com/denoland/deno_kv_oauth#getting-started-with-fresh

Reply:
Update: [this link](https://github.com/denoland/deno_kv_oauth#get-started-with-fresh) now guides users in creating their own plugin. Also, note that the `deno_fresh_kv_oauth` plugin is a third-party plugin, as the official Deno KV OAuth Fresh plugin has been removed.